tests/utils/aws_s3.go

```diff
@@ -0,0 +1,90 @@
+package utils
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func AwsS3CreateFolder(bucketName string, folderPath string) bool {
+	file := strings.NewReader("")
+	// Make sure escape('/') is at both ends of the path
+	folderPath = "/" + strings.TrimPrefix(folderPath, "/")
+	folderPath = strings.TrimSuffix(folderPath, "/") + "/"
+	filePath := folderPath + ".tmp"
+
+	sess, _ := session.NewSession(&aws.Config{
+		Region: aws.String("us-west-2"),
+	})
+
+	// Setup the S3 Upload Manager.
+	uploader := s3manager.NewUploader(sess)
+
+	// Upload the demo file to S3 bucket which will create a folder
+	_, err := uploader.Upload(&s3manager.UploadInput{
+		Bucket: aws.String(bucketName),
+		Key:    aws.String(filePath),
+		Body:   file,
+	})
+	if err != nil {
```

Comment:
let's return `err` instead, so the caller can handle it instead of the boolean value

Reply:
Changed return type to `error`.